src/optimizer/stats/cost.cpp

```diff
@@ -74,7 +74,7 @@ void Cost::CombineConjunctionStats(const std::shared_ptr<TableStats> &lhs,
       break;
     default:
       LOG_TRACE("Cost model conjunction on expression type %d not supported",
-                type);
+                (int)type);
```

Comment:
Change this to `LOG_WARN`.
Also use `ExpressionTypeToString()` instead of printing out the type integer value.